debugger_implementation.tex

```diff
@@ -158,19 +158,18 @@ \subsection{Using System Bus Access} \label{deb:mrsysbus}
     \hline
     Op & Address & Value & Comment \\
     \hline
-    Write & \Rsbaddresszero & address & \\
+    Write & \Rsbcs & \Fsbaccess$=2$, \Fsbreadonaddr, \Fsbreadondata, \Fsbautoincrement &
```

Comment:
In the generated PDF the 'Value' column overflows.

Reply:
Fixed.